test/spec.yml

```diff
@@ -864,10 +868,12 @@
     -   in: bn create node8 perplexingpersimmon
     -   in: bn deploy perplexingpersimmon
         err: |-
+            DebugId: *
```

Comment:
I am not strongly against it, but the issue I see with adding `DebugId` to every error coming from the system is that most errors are user errors which we can't provide support for: if users have a typo in their key we can't know what the typo was, if user code size is 300MB we can't tell them what can be deleted, etc.